clang/docs/ThreadSafetyAnalysis.rst

```diff
@@ -764,12 +764,6 @@ doesn't know that munl.mu == mutex.  The SCOPED_CAPABILITY attribute handles
 aliasing for MutexLocker, but does so only for that particular pattern.
 
 
-ACQUIRED_BEFORE(...) and ACQUIRED_AFTER(...) are currently unimplemented.
--------------------------------------------------------------------------
-
-To be fixed in a future update.
-
-
```

Comment:
They are implemented, but still under `-Wthread-safety-beta` as far as I know, because there are some weaknesses. If you want to touch this here, I would suggest to write instead that support is still experimental.

Reply:
Thanks! I documented those are "experimental".